.dependabot/config.yml

```diff
@@ -4,9 +4,9 @@ update_configs:
     directory: "/"
     update_schedule: "daily"
     ignored_updates:
-      # As dependabot doesn't currently support custom npm tags we exlude all bpk updates
-      # so that they don't bump non css versions
-      - match:
-          dependency_name: "bpk*"
+      # Once we are using css packages, and as long as dependabot doesn't support custom npm tags,
+      # we'll exlude all bpk updates so that they don't bump non css versions.
```

Comment:
Mis-spelling `exclude`

Reply:
@olliecurtis I think @shaundon has hacked you and is posting review comments under your account 😂

Reply:
:![bitmoji](https://sdk.bitmoji.com/render/panel/c6a53190-662b-4911-a95f-ddda7d526d2d-fc7ff7f2-599d-4581-9582-47fa255fb3ae-v1.png?transparent=1&palette=1&width=246)